src/screens/CreateProduct/styles.js

```diff
@@ -0,0 +1,98 @@
+import styled from 'styled-components/native';
+import {Container as C} from './../../styles/utils';
+import {darkblue} from './../../styles/colors';
+
+export const Container = styled(C)`
+  flex: 1;
+  padding: 20px 20px;
+  align-items: flex-start;
+  justify-content: flex-start;
+  background-color: white;
+`;
+
+export const Title = styled.Text`
+  font-size: 18px;
+  font-weight: bold;
+  margin-bottom: 10px;
+`;
+
+export const Group = styled.View`
+  width: 100%;
+  margin-top: 15px;
+`;
+
+export const Label = styled.Text`
+  font-size: 16px;
+  color: ${darkblue};
+  margin-bottom: 7px;
+`;
+
+export const Input = styled.TextInput`
+  background-color: #f5f5f5;
+  width: 100%;
+  border-radius: 5px;
+  padding: 10px 15px;
+  color: #828282;
+  font-size: 16px;
+`;
```

Comment:
Já existe um component de Input com os estilos padrão do layout. Você modificar ele se for necessário. 
Por exemplo como você fez com o componente de Container na linha 5.

Reply:
Legal